### PR TITLE
add --format option that uses standard-format to rewrite files

### PR DESCRIPTION
**Pull request by Flet**

### CODE REVIEW

File: bin/cmd.js [hunk 1]
```diff
@@ -5,10 +5,12 @@ var standard = require('../')
 
 var argv = minimist(process.argv.slice(2), {
   alias: {
+    format: 'F',
```
Comment (feross) [inline bin/cmd.js]: why not a lowercase `f`?

Comment (feross) [inline bin/cmd.js]: because it's a potentially destructive operation?

Comment (Flet) [inline bin/cmd.js]: Yep, exactly!

**Comment by feross** [comment]
Thanks, released as 2.7.0.

**Pull request merged**